homeassistant/components/hive/__init__.py

```diff
@@ -119,6 +119,7 @@ async def hot_water_boost(service):
         return False
 
     hass.data[DATA_HIVE] = hive
+    hass.data[DOMAIN]["entity_lookup"] = {}
```

Comment:
Please store all things under a single top level key in `hass.data`. Under that key and dict we can have several nested dicts etc as needed.